app/models/concerns/attachmentable.rb

```diff
@@ -63,7 +63,12 @@ def appropriate_extension(attachment)
   end
 
   def calculated_content_type(attachment)
-    content_type = Paperclip.run('file', '-b --mime :file', file: attachment.queued_for_write[:original].path).split(/[:;\s]+/).first.chomp
+    content_types = Paperclip.run('file', '-b -k -r --mime :file', file: attachment.queued_for_write[:original].path).split("\n- ")
+    content_types -= ['application/x-font-gdos']
```

Comment:
I would feel better if you aligned the `=` and put the mime type into a constant variable with a self-explaining name like `BOGUS_MIME_TYPES` or something.

Reply:
Addressed this just in case, but I think this PR isn't needed anymore, as the underlying bug in `file` has been fixed along with security fixes a while ago.

Reply:
Ah, in that case the PR can be closed. The workaround is still concerning to me because what if someone tried to upload a real font file, it would then be accepted and treated as MP3...

Reply:
It probably wouldn't, as the font file would probably not be detected as mp3 (the bug happens because both the mp3 magic and the font thing are detected, and the font thing used to take precedence even though it was unreliable and uses of that font format are incredibly rare).
Closing.